VultisigApp/VultisigApp/Extensions/TextField.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  TextField.swift
+//  VultisigApp
+//
+//  Created by Enrique Souza Soares on 30/06/24.
+//
+// This file extension is used to limit the number of characters that can be entered in a TextField.
+
+import Foundation
+import SwiftUI
+import Combine
+
+#if canImport(UIKit)
+import UIKit
+
+private var maxLengthKey: UInt8 = 0
+
+extension UITextField {
```

Comment:
This extension is unused. Let's remove as we're not using UITextFields from UIKit

Reply:
It is used to set max length to both macOs and IOs.

Reply:
You are right.